### DISCUSSION TITLE
claude-3-sonnet-20240229 is not supported on this API. Please use the Messages API instead.

### DISCUSSION THREAD

**Discussion by mbaboo** [danny-avila/LibreChat#2019] [Troubleshooting] [answered]
I upgraded to the latest version of the code and enabled one of the new claude-3 models in my .env file. This model is listed in the .env.example file. 

I got this error:

Something went wrong. Here's the specific error message we encountered: {"type":"error","error":{"type":"invalid_request_error","message":"\"claude-3-sonnet-20240229\" is not supported on this API. Please use the Messages API instead."}}

The older claude models (eg. claude-instant-1.2) work fine. 

Presumably LibreChat is using the Text Completions API and not the Messages API.

**Comment by fuegovic** [comment, marked as answer]
the update probably did not apply properly, look here for the same problem and a possible solution: https://github.com/danny-avila/LibreChat/discussions/1988#discussioncomment-8700351

**Reply by mbaboo** [reply]
Thanks. It works now. 

For anyone else coming here with the same problem, here is what I did to fix the problem. Of course if you have a different setup you might have to run slightly different commands but the approach would probably be the same. 

Stop the container.

`sudo docker-compose down`


Remove the LibreChat Service. Edit docker-compose.yml. Look for the section related to LibreChat and remove it entirely. Save the file. 

`sudo nano docker-compose.yml`

Remove cached images.

`sudo docker image prune -a`


Remove dangling images.

`sudo docker image prune`

Add back the LibreChat Service. Delete the changed docker-compose.yml and bring back the latest version.

```
rm docker-compose.yml
git checkout -- docker-compose.yml
```

Rebuild and restart.

```
sudo docker-compose build 
sudo docker-compose up -d
```
